addon/services/log.js

```diff
@@ -396,6 +424,28 @@ export default Ember.Service.extend(Ember.Evented, {
     this.set('_originalMethodsCache', originalMethodsCache);
   },
 
+  /**
+   * Initializes properties of a log service.
+   */
+  initProperties() {
+    let config = getOwner(this).resolveRegistration('config:environment');
+    let logConfiguration = config.APP.log;
+
+    this.set('enabled', Ember.typeOf(logConfiguration.enabled) === 'boolean' && logConfiguration.enabled);
+    this.set('storeErrorMessages', Ember.typeOf(logConfiguration.storeErrorMessages) === 'boolean' && logConfiguration.storeErrorMessages);
+    this.set('storeWarnMessages', Ember.typeOf(logConfiguration.storeWarnMessages) === 'boolean' && logConfiguration.storeWarnMessages);
+    this.set('storeLogMessages', Ember.typeOf(logConfiguration.storeLogMessages) === 'boolean' && logConfiguration.storeLogMessages);
+    this.set('storeInfoMessages', Ember.typeOf(logConfiguration.storeInfoMessages) === 'boolean' && logConfiguration.storeInfoMessages);
+    this.set('storeDebugMessages', Ember.typeOf(logConfiguration.storeDebugMessages) === 'boolean' && logConfiguration.storeDebugMessages);
+    this.set('storeDeprecationMessages', Ember.typeOf(logConfiguration.storeDeprecationMessages) === 'boolean' && logConfiguration.storeDeprecationMessages);
+    this.set('storePromiseErrors', Ember.typeOf(logConfiguration.storePromiseErrors) === 'boolean' && logConfiguration.storePromiseErrors);
+    this.set('showPromiseErrors', Ember.typeOf(logConfiguration.showPromiseErrors) === 'boolean' && logConfiguration.showPromiseErrors);
+
+    if (Ember.typeOf(logConfiguration.applicationLogModelName === 'string')) {
```

Comment:
The bracket ran away:
```suggestion
    if (Ember.typeOf(logConfiguration.applicationLogModelName) === 'string') {
```